### PR TITLE
brainbuilder -> 0.11.3

### PR DESCRIPTION
**Pull request by arsenius7**

### CODE REVIEW

File: var/spack/repos/builtin/packages/brainbuilder/package.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Brainbuilder(PythonPackage):
     git      = "ssh://bbpcode.epfl.ch/nse/brainbuilder"
 
     version('develop', branch='master')
-    version('0.11.2', tag='brainbuilder-v0.11.2', preferred=True)
+    version('0.11.3', tag='brainbuilder-v0.11.3', preferred=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/brainbuilder/package.py]:
```suggestion
    version('0.11.3', tag='brainbuilder-v0.11.3')
```
The newest version is always preferred.

**Review by matz-e — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/brainbuilder/package.py (hunk 1 above)
Comment (arsenius7) [reply var/spack/repos/builtin/packages/brainbuilder/package.py]: I vaguely recall there was a catch with lexicographical sort order where `0.9` turned out to be "newer" than `0.10`; but maybe it was with `module load`.

Comment (matz-e) [reply var/spack/repos/builtin/packages/brainbuilder/package.py]: You're also referencing the version quite explicitly during deployment…

Comment (arsenius7) [reply var/spack/repos/builtin/packages/brainbuilder/package.py]: Also true. ^^

**Review by matz-e — APPROVED** [review]

**Pull request merged**